### DISCUSSION TITLE
Welcome to Hoot Draft Discussions!

### DISCUSSION THREAD

**Discussion by mattheworres** [mattheworres/phpdraft#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  I'm hoping that Discussions is a better implementation than Gitter is, but will keep Gitter up for the time being until I get a better feel for how Discussions works.

Issues? Comment here, comment on Gitter, or hit me up on Twitter. Thanks!

- Matt

**Comment by zo0o0ot** [comment]
Nice! Lately, I've been doing some development using GitPod to ensure a consistent development environment.  I'd really appreciate if we could get either GitPod or GitHub Codespaces.
My install of Hoot Draft is currently broken, and I'd love to get things back up and running again.

**Comment by mattheworres** [comment]
Nice! I'm out of the loop, so I've never heard of GitPod or Codespaces, but it sounds like something pretty worthwhile.

Vagrant was a nice step forward from the "old" days of having to manually install all the dependencies, but it's still pretty heavyweight, and I was considering making the small jump to Docker to do basically the same thing, but GitPod might actually be an even better improvement. I'll look into it when I have time, or I can offer advice if this is something you're interested in doing (I know you're not as familiar with the codebase as I am, so it may be tough).

**Reply by zo0o0ot** [reply]
It sounds like the work with Docker could help with GitPod, and potentially vice versa.  I know that getting gitpod going requires a dockerfile.

**Reply by zo0o0ot** [reply]
Here's a link that has more info:
https://www.gitpod.io/blog/gitpodify/

I've set up Go and Hugo along with C# and .NET Core, but my use cases haven't needed databases yet.  It appears that there is info on how to do that, though.
https://www.gitpod.io/blog/gitpodify/#installing-databases
